packages/common/driver-definitions/src/storage.ts

```diff
@@ -128,9 +128,15 @@ export interface IDocumentStorageService extends Partial<IDisposable> {
      * @param versionId - Version id of the requested version.
      * @param count - Number of the versions to be fetched.
      * @param scenarioName - scenario in which this api is called. This will be recorded by server and would help
-     *  in debugging purposes to see why this call was made.
-     */
-    getVersions(versionId: string | null, count: number, scenarioName?: string): Promise<IVersion[]>;
+     * @param bypassCache - True to bypass cache for getting versions and get it from the server.
+    *  in debugging purposes to see why this call was made.
```

Comment:
You broke a comment for previous arg.

I'd add some small narative there in the form of
Driver may choose to cache requests and serve data from cache. That will result in stale info returned.
Callers can disable this functionality by passing bypassCache  = true and ensuring that driver will return latest information from storage.

Reply:
I think Vlad mentioned you broke the text: 
 in debugging purposes to see why this call was made.

From the original description.